app/Http/Controllers/ContestController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Gladiator\Http\Controllers;
+
+use Gladiator\Models\Contest;
+use Gladiator\Http\Requests\ContestRequest;
+
+class ContestController extends Controller
```

Comment:
yeah I noticed but `ContestsController` just sounded funny to me for some reason, I can change it if you think its better that way